css/style.css

```diff
@@ -0,0 +1,92 @@
+
+/*#fadeImage{
+  background: white;
+}*/
+
+
+#container{
+  display: none;
+  max-width: 960px;
+  margin: auto;
+}
+
+body{
+  background: url("space4.gif");
+}
+
+#page-header{
+  border:0;
+  box-sizing: border-box;
+  clear: both;
+  display: block;
+  float: left;
+  margin: auto;
+  padding: 2% 8% 2% 8%;
+  width: 100%;
+}
+
+.clearfix{
+  clear: both;
+}
+
+
+.nav li{
+  border: 2px white solid;
+  display: inline-block;
+  float: right;
+  margin: 2%;
+  padding: 1%;
+}
+
+.nav li:hover{
+  background: white;
+}
+
+.nav li a:hover{
+  color: black;
+}
```

Comment:
This is a cool effect - using the inverse of the background and font color. Nice use of pseudo selectors!